test/SerializerV1Test.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace AgDevelop\ObjectMirror\Tests;
+
+use AgDevelop\Interface\Json\SerializableInterface;
+use AgDevelop\ObjectMirror\Serializer\V1\Serializer;
+use AgDevelop\ObjectMirror\Tests\Sample\ExampleA;
+use AgDevelop\ObjectMirror\Tests\Sample\ExampleB;
+use AgDevelop\ObjectMirror\Tests\Sample\ParamDto;
+use PHPUnit\Framework\TestCase;
+
+class SerializerV1Test extends TestCase
+{
+    public function provideSerialize()
```

Comment:
Może `void`?

Reply:
Nie widzę innego sensownego sposobu jak tylko wymuszenie tego przez regułę w php-cs-fixerze.
Dodałem do php-cs-fixera regułę 'return_void', którą php-cs-fixer określa jako 'risky' bo zmienia sygnatury funkcji... ale być może we wszystkich nowych komponentach nie jest to dla nas risk.